.markdownlint.yaml

```diff
@@ -0,0 +1 @@
+MD013: false
```

Comment:
Are we sure we want to remove it completely? Can we touch modify it to use something more reasonable nowadays like 120 characters?

Reply:
I didn't want this in the PR, removing!